### DISCUSSION TITLE
Remove Slack login button for oss

### DISCUSSION THREAD

**Discussion by yoramshai** [outline/outline#4735] [Self Hosting] [answered]
Is there an option to remove the slack auth without removing slack integration?

**Comment by tommoor** [comment, marked as answer]
Find slack in the `authentication_providers` table and set `enabled=false`

**Reply by yoramshai** [reply]
Thank you so much for your answer.
looks like the slack is not there for some reason
<img width="644" alt="image" src="https://user-images.githubusercontent.com/11707803/213286790-27e48496-94c0-4b4e-a088-af1a01b877d7.png">

**Reply by tommoor** [reply]
In this case you could add it

**Reply by yoramshai** [reply]
unfortunately, after adding it still no luck 

<img width="694" alt="image" src="https://user-images.githubusercontent.com/11707803/213383951-bcb8e127-ae2b-483c-ae52-eb256a3088ec.png">

**Reply by tommoor** [reply]
Then I'm not sure, do you have multiple teams somehow?

**Reply by yoramshai** [reply]
Nope, only one.

**Reply by yoramshai** [reply]
This logic is not working. 
I set the google one to enabled=false, and I still got the google auth on the login page.

**Reply by tommoor** [reply]
Yes, it looks like this does not work the same in community edition – it seems like a bug

**Reply by yoramshai** [reply]
@tommoor thanks for confirming it.
Want me to open a bug?

**Reply by tommoor** [reply]
I created https://github.com/outline/outline/issues/4743 already
